blobmodel/blobs.py

```diff
@@ -71,20 +71,19 @@ def discretize_blob(
             return self._single_blob(
                 x_perp, y_perp, t, Ly, periodic_y, one_dimensional=one_dimensional
             )
-        if np.sin(self.theta) == 0:
-            x_border = Ly - self.pos_y
-            adjusted_Ly = Ly
+        if self.theta == 0:
+            number_of_y_propagations = 0
         else:
             x_border = (Ly - self.pos_y) / np.sin(self.theta)
             adjusted_Ly = Ly / np.sin(self.theta)
-        if type(t) in [int, float]:
             # t has dimensionality = 0, used for testing
```

Comment:
In the current form it isn't obvious where this comment is referring to. We should make clear this comment refers to the if type(t) line

Reply:
chnaged